psi4/driver/json_wrapper.py

```diff
@@ -50,9 +49,133 @@
 
 
 def run_json(json_data):
+
+    # Set scratch
+    if "scratch_location" in json_data:
+        psi4_io = core.IOManager.shared_object()
+        psi4_io.set_default_path(json_data["scratch_location"])
+
+    # Set memory
+    if "memory" in json_data:
+        psi4.set_memory(json_data["memory"])
+
+    # Do we return the output?
+    return_output = json_data.pop("return_output", False)
+    if return_output:
+        outfile = os.path.join(core.IOManager.shared_object().get_default_path(), str(uuid.uuid4()) + ".json_out")
+        core.set_output_file(outfile, False)
+        json_data["raw_output"] = "Not yet run."
+
+    # Set a few flags
+    json_data["raw_output"] = None
+    json_data["success"] = False
+
+    # Attempt to run the computer
+    try:
+        if ("schema_name" in json_data) and (json_data["schema_name"] == "QC_JSON"):
+            run_json_qc_schema(json_data)
+        else:
+            run_json_original_v1_1(json_data)
+
+    except Exception as error:
+        json_data["error"] = repr(error)
+        json_data["success"] = False
+
+    if return_output:
+        with open(outfile, 'r') as f:
+            json_data["raw_output"] = f.read()
+        os.unlink(outfile)
+
+    return json_data
+
+
+def run_json_qc_schema(json_data):
+    """
+    An implementation of the QC JSON Schema (molssi-qc-schema.readthedocs.io/en/latest/index.html#) implementation in Psi4.
+
+
+    Parameters
+    ----------
+    json_data : JSON
+        Please see molssi-qc-schema.readthedocs.io/en/latest/spec_components.html for further details.
+
+    Notes
+    -----
+    !Warning! This function is experimental and likely to change in the future.
+    Please report any suggestions or uses of this function on github.com/MolSSI/QC_JSON_Schema.
+
+    Examples
+    --------
+
+    """
+
+    # This is currently a forced override
+    json_data["schema_version"] = "v0.1"
```

Comment:
Note that I had to change this from `0` to correctly ID the schema.

Reply:
Hmm, I thought we had discussed going to an integer based scheme so that parsers were not required to parse versioning semantics?

Reply:
Ah yes, so we did. In that case, I should have altered the test cases. Will switch out.